openfecwebapp/app.py

```diff
@@ -40,7 +40,7 @@ def get_absolute_url():
 
 def get_canonical_url():
     path = furl.furl(request.url).path
-    return '{}/data{}'.format(config.canonical_base, path)
+    return '{}{}'.format(config.canonical_base, path)
```

Comment:
Just confirming we don't need the `/data` here - I know we changed some things around this but wanted to be sure!